src/thermopro_ble/parser.py

```diff
@@ -25,7 +25,14 @@
 }
 
 UNPACK_TEMP_HUMID = Struct("<hB").unpack
-UNPACK_SPIKE_TEMP = Struct("<h").unpack
+UNPACK_SPIKE_TEMP = Struct("<BHHH").unpack
+
+MULTIPROBE_DEVICES = {
+    "TP962R": {
+        0: "black",
+        1: "white",
+    },
```

Comment:
I'd keep these as probe 0 and probe 1 since the vendor tends to add new devices with the same format and we can't be sure they will keep the color scheme in the future

Reply:
I'm actually already falling back to something like this on lines 85-92, I just use the color names when we know that the model has a specific mapping.  The difference is for probe 0 I default to no affix at all.

Since we don't know if any new model has more than one probe, the alternative here would be to have "Probe 0" affixed even on single probe models.  I think this is reasonable also, and would simplify the code.  I started a review for that option below.